src/Compute/Compute/Generated/ContainerService/ContainerServiceDeleteMethod.cs

```diff
@@ -48,7 +48,16 @@ public override void ExecuteCmdlet()
                     string resourceGroupName = this.ResourceGroupName;
                     string containerServiceName = this.Name;
 
-                    var result = ContainerServicesClient.DeleteWithHttpMessagesAsync(resourceGroupName, containerServiceName).GetAwaiter().GetResult();
+                    Microsoft.Rest.Azure.AzureOperationResponse result;
+                    if (NoWait.IsPresent)
+                    {
+                        result = ContainerServicesClient.BeginDeleteWithHttpMessagesAsync(resourceGroupName, containerServiceName).GetAwaiter().GetResult();
+                    }
+                    else
+                    {
+                        result = ContainerServicesClient.DeleteWithHttpMessagesAsync(resourceGroupName, containerServiceName).GetAwaiter().GetResult();
```

Comment:
This is another one that uses GetOperationIdFromUrlString, which may not work with the Begin* method, let's leave this one out as well.